hotsos/defs/scenarios/storage/ceph/ceph-rgw/bugs/lp1974138.yaml

```diff
@@ -0,0 +1,24 @@
+checks:
+  node_is_ceph_rgw_and_beast_supported_version:
+    apt:
+      librgw2:
+        - min: 15.2.0
+  ssl_enabled:
+    property: hotsos.core.plugins.openstack.OpenstackBase.ssl_enabled
+  apache2_allow_encoded_slashes_on:
+    property:
+      path: hotsos.core.plugins.openstack.OpenstackBase.apache2_allow_encoded_slashes_on
+      ops: [[eq, false]]
```

Comment:
You don't need to compare boolean property. You can simply drop this line and use "not" at the `conclusions` like:

```
      - node_is_ceph_rgw_and_beast_supported_version
      - ssl_enabled
      - not: apache2_allow_encoded_slashes_on
```